### ISSUE TITLE
It crashes on the logo on my realme gt 2 pro..

### ISSUE THREAD

**Issue by Manthanxyz** [LagradOst/CloudStream-3#960]
**Guidelines:**
- [ ] It **is not** a duplicate issue.
- [ ] If related to a provider, I have checked the site and it works, but not the app.

**To Reproduce**

**Expected behavior**
It sticks on the cloudstream logo for a few secs and then crashes

**Device **Realme Gt 2 pro
 - Android Version: [Android 12] .

**Comment by LagradOst** [comment]
Have you tested the latest pre release

**Comment by Manthanxyz** [comment]
Yeah..I got a option from the app to update to the latest version..after
that it began to crash.
